source/user-manual/agents/agent-connection.rst

```diff
@@ -3,50 +3,164 @@
 .. meta::
   :description: Find out more about how to check the connection to the Wazuh Manager in this section of our documentation. 
   
-.. _agent-connection:
+Checking connection with the Wazuh manager
+==========================================
 
-Checking connection with Manager
-================================
+Using the Wazuh dashboard
+-------------------------
 
-Before you check the agent's connection with the manager, first ensure the agent is pointing to the manager IP address. This is set in ``ossec.conf`` using the ``<client>`` XML tag. For more on this, see :ref:`Client reference <reference_ossec_client>`.
+You can check the connection status of any agent selecting the **Agents** menu option of the Wazuh dashboard.
 
-.. code-block:: xml
+.. thumbnail:: /images/manual/managing-agents/agents-menu.png
+   :title: Wazuh dashboard agents menu option
+   :align: center
+   :width: 80%
 
-  <ossec_config>
-    <client>
-      <server>
-        <address>10.0.0.10</address>
-        <protocol>tcp</protocol>
-      </server>
-    </client>
-  </ossec_config>
+This option shows the **Agents** dashboard with a list of all registered agents. This list includes the connection status of each agent. This dashboard also shows the amount of agents found for each possible connection status.
 
-This will set 10.0.0.10 as the Wazuh server. Once this is done, you will need to restart the Agent:
+.. thumbnail:: /images/manual/managing-agents/agents-dashboard.png
+   :title: Wazuh dashboard agents menu option
+   :align: center
+   :width: 80%
 
-  a. For Systemd:
+Reading the local `wazuh-agentd.state` file
+-------------------------------------------
 
-    .. code-block:: console
+You can read the :doc:`wazuh-agentd.state </user-manual/reference/statistics-files/wazuh-agentd-state>` file found in the endpoint to check the status of the connection. The Wazuh agent keeps reporting its connection status in this file as follows.
 
-      # systemctl restart wazuh-agent
+-  ``pending``: Waiting for acknowledgement from the Wazuh manager about connection established.
+-  ``disconnected``: No acknowledgement signal received during the last 60 seconds or lost connection.
```

Comment:
The spelling of acknowledgement is a non-American variant. For consistency, consider replacing it with the American English spelling.